src/test/java/org/assertj/core/api/assumptions/Assumptions_assumeThat_with_extracting_Test.java

```diff
@@ -63,7 +63,7 @@ public void should_run_test_when_assumption_using_extracting_on_list_passes() {
 
   @Test
   public void should_run_test_when_assumption_using_extracting_on_object_passes() {
-    assertThatCode(() -> assumeThat(yoda).extracting("name").containsExactly("Yoda")).doesNotThrowAnyException();
+    assertThatCode(() -> assumeThat(yoda).extracting("name").isEqualTo("Yoda")).doesNotThrowAnyException();
   }
```

Comment:
add a new test checking `extracting` with multiple arguments since it is now missing

Reply:
I was not sure how to update this test because it is targeting an `Iterable` and from my point of view the `Map` was not really relevant here (also because the `extracting` of `AbstractIterableAssert` is never reached in case of maps). Since I was changing the `extracting` in `AbstractMapAssert` this test was also affected, therefore my quick fix was to wrap the `Map`.

Instead of adding another test here, I would suggest to remove the `Map` instance and to use any `Iterable` instead. What do you think?